rules_java_gapic/resources/gradle/grpc.gradle.tmpl

```diff
@@ -17,12 +17,12 @@ dependencies {
 }
 
 task javadocJar(type: Jar) {
-  classifier = 'javadoc'
+  archiveClassifier = 'javadoc'
```

Comment:
What's the minimum Gradle version that supports `archiveClassifier`? This file is used by self-service client libraries, if a customer is using an very old version of Gradle, I guess it would not work anymore for them?

Reply:
Tried 8.1 but didn't work. As suggested, I have separated out build (21) job and excluded the self service clients steps in it. Does that work?